src/components/TestToolMenu.js

```diff
@@ -45,8 +45,10 @@ const TestToolMenu = props => (
         This enables QA and internal testers to take advantage of sandbox environments for 3rd party services like Plaid and Onfido. */}
```

Comment:
The comment above here is not correct @kidroca, can you remove the "secure" part from the line above as this is changing not only secure api but any api. Thanks!